test/lib/cache.js

```diff
@@ -45,6 +45,7 @@ const setupCacacheFixture = () => {
     'webpack@4.44.1',
     'npm@1.2.0',
     'webpack@4.47.0',
+    'foo@1.2.3-beta',
```

Comment:
let's also represent a package with a `-` with a semver both with and without a `-`

Reply:
we should also have scoped packages in here

Reply:
added tests